src/OrchardCore.Modules/OrchardCore.Resources/LocalResourceConfigureOptions.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.Resources;
+
+public class LocalResourceConfigureOptions : IConfigureOptions<ResourceOptions>
+{
+    private readonly IConfiguration _configuration;
+
+    public LocalResourceConfigureOptions(IConfiguration configuration)
+    {
+        _configuration = configuration;
+    }
+
+    public void Configure(ResourceOptions options)
+    {
+        _configuration.GetSectionCompat("OrchardCore:Site_Settings").Bind(options);
```

Comment:
Here I would have named the section `OrchardCore_Resources` in place of `Site_Settings`.

Reply:
Updated